library/koji_tag.py

```diff
@@ -182,6 +182,8 @@ def ensure_inheritance(session, tag_name, tag_id, check_mode, inheritance):
         maxdepth = rule.get('maxdepth')
         if maxdepth == '':
             maxdepth = None
+        if isinstance(maxdepth, str):
```

Comment:
Maybe we should use `from ansible.module_utils.six import text_type` here instead of just `str`.